examples/05_glm_second_level/plot_second_level_one_sample_test.py

```diff
@@ -107,32 +129,57 @@
 # we make no false discovery at all).  This threshold is much more conservative
 # than the previous one.
 threshold = 1
-title = ('Group left-right button press: \n'
-         'parametric test (FWER < 10%)')
-display = plotting.plot_glass_brain(
-    neg_log_pval, colorbar=True, display_mode='z', plot_abs=False, vmax=3,
-    cut_coords=cut_coords, threshold=threshold, title=title)
-plotting.show()
 
-###########################################################################
+###############################################################################
 # Now, we compute the (corrected) p-values with a permutation test.
 from nilearn.glm.second_level import non_parametric_inference
-neg_log_pvals_permuted_ols_unmasked = \
-    non_parametric_inference(second_level_input,
-                             design_matrix=design_matrix,
-                             model_intercept=True, n_perm=1000,
-                             two_sided_test=False,
-                             smoothing_fwhm=8.0, n_jobs=1)
-
-###########################################################################
+
+out_dict = non_parametric_inference(
+    second_level_input,
+    design_matrix=design_matrix,
+    model_intercept=True,
+    n_perm=1000,
+    two_sided_test=False,
+    smoothing_fwhm=8.0,
+    n_jobs=1,
+    threshold=0.001,
+)
+
```

Comment:
The example now lasts several minutes on my box.  My impression is that it is significantly longer now. I'd do 3 things:
* state explicitly that performing cluster-level inference has a cost in terms of computation time
* (as for tfce) reduce the number of permutations (explaining that this is bad)
* explain that  n_jobs can be chosen to reduce cpu time.